pkg/kev/converter/kubernetes/utils_test.go

```diff
@@ -609,8 +609,7 @@ var _ = Describe("Utils", func() {
 			Context("containing kev specific labels", func() {
 				projectService, err := NewProjectService(composego.ServiceConfig{
 					Labels: composego.Labels{
-						"FOO":                   "BAR",
-						config.LabelServiceType: "value",
+						"FOO": "BAR",
```

Comment:
It'd be good to replace that (removed) label with another one for the time being? Otherwise this context doesn't make much sense any more as it's about handling kev specific config labels.

Reply:
This is about filtering invalid labels and maintaining the rest. There is no other label to put there since they are all going away but the labels are still part of the spec and AIUI they can still be provided even if we don't use them?

Reply:
Yeh, when labels go away eventually this will become obsolete as there will be no need for filtering labels anymore. At the moment I think there is still a way to provided labels that haven't been ported yet which we could use?